prow/cmd/phaino/README.md

```diff
@@ -24,6 +24,9 @@ bazel run //prow/cmd/mkpj -- --config-path=/your/config --job=foo > /tmp/foo
 bazel run //prow/cmd/phaino -- /tmp/foo
 ```
 
+Phaino is an interactive utility; it will prompt you for a local copy of any secrets or
```

Comment:
There are plans to add flags for this so you could do somthing like `--secret=agent=/etc/bond`